internal/handlers/pg/pgdb/transactions.go

```diff
@@ -109,12 +108,11 @@ func (pgPool *Pool) InTransactionRetry(ctx context.Context, f func(pgx.Tx) error
 	// TODO use exponential backoff with jitter instead
 	// https://github.com/FerretDB/FerretDB/issues/1720
```

Comment:
I think we could remove that comment now :) Please check other comments like this one

Reply:
removed